packages/rum/src/domain/record/observers.spec.ts

```diff
@@ -328,3 +333,84 @@ describe('initStyleSheetObserver', () => {
     })
   })
 })
+
+describe('initMouseInteractionObserver', () => {
+  let mouseInteractionCallbackSpy: jasmine.Spy<MouseInteractionCallBack>
+  let stopObserver: () => void
+  let sandbox: HTMLDivElement
+  let a: HTMLAnchorElement
+  let coordinatesComputed: boolean
+
+  beforeEach(() => {
+    if (isIE()) {
+      pending('IE not supported')
+    }
+    if (!window.visualViewport) {
+      pending('no visualViewport')
+    }
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏those tests should pass also on browsers that lacks visual viewport support. Whether `visualViewport` is used is an implementation detail.

Reply:
My intent with those tests was to ensure that blur events was not accessing visualViewort API since it is the one that trigger a forced layout and is related to the safari bug and prevent implementation s where coordinates were computed for those events but dropped before generating the record.
In cases where there is no visualViewport API, there is no issue.

I could clarify the intent in the test by replacing `compute x/y coordinates` references by `force layout`, wdyt?

Reply:
Mmh I would keep general tests on `initMouseInteractionObserver` and add a section specifically for the `force layout` issue that is run only on browsers supporting visualviewport.

Reply:
Sounds good to me